### DISCUSSION TITLE
Public list RSS feed

### DISCUSSION THREAD

**Discussion by perkfly** [community/community#8769] [Other Feature Feedback, Questions, & Ideas]
Provide a RSS/Atom feed for public list so that people can follow up on them

**Comment by brianlovin** [comment]
Love it, thanks for the suggestion @yifeikong — I'll add your +1 to our tracking issue for more ways to follow along with changes to a list

**Comment by tkapias** [comment]
I would like to support this Feature.

I am currently using a script that create an OPML file from the Releases page of each Starred project in a list. Then I import this OPML file into Miniflux from time to time.

What would be perfect would be to generate an RSS feed per List that directly gives the Releases information for each project.

Or, at least a button to generate an OPML file per list containing the feed for each Release pages.

**Comment by BluejacketScott** [comment]
Just found this yesterday. I think it might just do the trick (or at least be a temp solution) and is [selfhosted](https://github.com/topics/self-hosted) - 
Courtesy of [DIYGod](https://github.com/DIYgod):

[**RSSHub** Introduction](https://github.com/DIYgod/RSSHub#introduction)

> RSSHub is an open source, easy to use, and extensible RSS feed generator. It's capable of generating RSS feeds from pretty much everything.

[RSSHub](https://github.com/DIYgod/RSSHub#introduction)
